monkestation/code/modules/gauntlet/syndie.dm

```diff
@@ -0,0 +1,245 @@
+
+/obj/item/badmin_stone/syndie
+	name = "Syndie Stone"
+	desc = "Power, baby. Raw power."
+	color = "#ff0130"
+	force = 30
+	stone_type = SYNDIE_STONE
+	ability_text = list("ALL INTENTS: PLACEHOLDER MARTIAL ART")
+	spell_types = list(
+		/datum/action/spell/self/infinity/regenerate,
+		/datum/action/spell/self/infinity/syndie_bullcharge,
+		/datum/action/spell/self/infinity/syndie_jump)
+	gauntlet_spell_types = list(
+		/datum/action/spell/self/infinity/shockwave/syndie_stone)
+	var/datum/martial_art/cqc/martial_art
+
+/obj/item/badmin_stone/syndie/Initialize()
+	. = ..()
+	martial_art = new
+
+/obj/item/badmin_stone/syndie/HelpEvent(atom/target, mob/living/user, proximity_flag)
+	if(ishuman(user) && ishuman(target) && proximity_flag)
+		martial_art.help_act(user, target)
+
+/obj/item/badmin_stone/syndie/DisarmEvent(atom/target, mob/living/user, proximity_flag)
+	if(ishuman(user) && ishuman(target) && proximity_flag)
+		martial_art.disarm_act(user, target)
+
+/obj/item/badmin_stone/syndie/HarmEvent(atom/target, mob/living/user, proximity_flag)
+	if(ishuman(user) && ishuman(target) && proximity_flag)
+		martial_art.harm_act(user, target)
+
+/obj/item/badmin_stone/syndie/GrabEvent(atom/target, mob/living/user, proximity_flag)
+	if(ishuman(user) && ishuman(target) && proximity_flag)
+		martial_art.grab_act(user, target)
+
+/obj/item/badmin_stone/syndie/GiveAbilities(mob/living/L, gauntlet = FALSE)
+	. = ..()
+	if(ishuman(L))
+		martial_art.teach(L)
+	ADD_TRAIT(L, TRAIT_THERMAL_VISION, SYNDIE_STONE_TRAIT)
+
+/obj/item/badmin_stone/syndie/RemoveAbilities(mob/living/L, gauntlet = FALSE)
+	. = ..()
+	if(ishuman(L))
+		martial_art.remove(L)
+	REMOVE_TRAIT(L, TRAIT_THERMAL_VISION, SYNDIE_STONE_TRAIT)
+
+/////////////////////////////////////////////
+/////////////////// SPELLS //////////////////
+/////////////////////////////////////////////
+
+/datum/action/spell/self/infinity/shockwave/syndie_stone
+	name = "Syndie Stone: Shockwave"
+	background_icon = 'monkestation/icons/obj/infinity.dmi'
+	background_icon_state = "syndie"
+	range = 8
+
+/datum/action/spell/self/infinity/regenerate
+	name = "Syndie Stone: Regenerate"
+	desc = "Regenerate 4 health per second. Requires you to stand still."
+	button_icon_state = "regenerate"
+	background_icon = 'monkestation/icons/obj/infinity.dmi'
+	background_icon_state = "syndie"
+	stat_allowed = TRUE
+
+/datum/action/spell/self/infinity/regenerate/cast(list/targets, mob/user)
+	if(isliving(user))
+		var/mob/living/L = user
+		if(L.on_fire)
+			to_chat(L, "<span class='notice'>The fire interferes with your regeneration!'</span>")
+			revert_cast(L)
+			return
+		if(L.stat == DEAD)
+			to_chat(L, "<span class='notice'>You can't regenerate out of death.</span>")
+			revert_cast(L)
+			return
+		while(do_after(L, 10, FALSE, L))
+			L.visible_message("<span class='notice'>[L]'s wounds heal!</span>")
+			L.heal_overall_damage(4, 4, 4, null, TRUE)
+			L.adjustToxLoss(-4)
```

Comment:
```suggestion
			L.adjustToxLoss(-4, forced = TRUE)
```
as this a stone spell rather than thanos-specific, there's the possibility of oozelings using it.